public/manifest.json

```diff
@@ -25,6 +25,6 @@
   ],
   "start_url": ".",
   "display": "standalone",
-  "theme_color": "#16385c",
-  "background_color": "#ffffff"
+  "theme_color": "#20293a",
+  "background_color": "#20293a"
```

Comment:
Are you sure?
> The background_color should be the same color as the load page, to provide a smooth transition from the splash screen to your app

https://developers.google.com/web/fundamentals/web-app-manifest/#splash-screen

Reply:
Twitter's splash screen in blue. App background is white. It's a matter of choice.